tests/karate/src/test/java/fe/net/ReplyMethods.java

```diff
@@ -0,0 +1,31 @@
+package fe.net;
+
+import com.intuit.karate.Json;
+import common.net.MessageBuffer;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+/**
+ * This class contains useful message replies that can be used when using {@link
+ * MockBackend#setReplyProducer(Function)}
+ */
+public class ReplyMethods {
+
+  private static final String VALID_REPLY_TEMPLATE =
+      "{\"jsonrpc\":\"2.0\",\"id\":%ID%,\"result\":0}";
+  private static final String VALID_CATCHUP_REPLY_TEMPLATE =
+      "{\"jsonrpc\":\"2.0\",\"id\":%ID%,\"result\":[]}";
+
```

Comment:
What about error templates ?

Reply:
No test need it yet so I did not implement the error template yet. But when a new test needs it, we can create the template easily.